runtime/common/src/lib.rs

```diff
@@ -164,21 +165,25 @@ pub mod asset_registry {
 }
 
 pub mod changes {
+	use cfg_primitives::SECONDS_PER_WEEK;
+	use cfg_types::fixed_point::Ratio;
 	use codec::{Decode, Encode, MaxEncodedLen};
 	use frame_support::RuntimeDebug;
 	use pallet_loans::entities::changes::Change as LoansChange;
+	use pallet_pool_fees::types::Change as PoolFeesChange;
 	use pallet_pool_system::pool_types::changes::PoolChangeProposal;
 	use scale_info::TypeInfo;
 	use sp_runtime::DispatchError;
 
 	#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
-	pub enum RuntimeChange<T: pallet_loans::Config> {
+	pub enum RuntimeChange<T: pallet_loans::Config + frame_system::Config> {
 		Loan(LoansChange<T>),
+		PoolFee(PoolFeesChange<<T as frame_system::Config>::AccountId, Ratio>),
 	}
 
 	#[cfg(not(feature = "runtime-benchmarks"))]
-	impl<T: pallet_loans::Config> From<RuntimeChange<T>> for PoolChangeProposal {
-		fn from(RuntimeChange::Loan(loans_change): RuntimeChange<T>) -> Self {
+	impl<T: pallet_loans::Config + frame_system::Config> From<RuntimeChange<T>> for PoolChangeProposal {
```

Comment:
You've done all this part perfectly! 🙌🏻 

I'm adding also a new `RuntimeChange` for oracles. I think we can do something like the following to handle the amount of bounds easier:
```rust
	/// Auxiliar type to carry all pallets that use RuntimeChange
	pub trait Changeable: pallet_loans::Config + pallet_oracle_data_collection::Config {}
	impl<T: pallet_loans::Config + pallet_oracle_data_collection::Config> Changeable for T {}
```

And later use `Changeable` in every bound list, to reduce the boilerplate, WDYT?

For `From` and `TryInto` impl I'm considering even a macro to reduce the boilerplate

Reply:
> And later use `Changeable` in every bound list, to reduce the boilerplate, WDYT?
> 
> For `From` and `TryInto` impl I'm considering even a macro to reduce the boilerplat

Both of that sounds like a great improvement to me!